docs/release-notes/updated-user-roles-slot-caps.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  WebUI: In the enterprise edition of Determined, give Viewer, Editor, GenAI, and Workspace Admin
```

Comment:
We also gave the role to the `EditorRestricted` and `EditorProjectRestricted` roles, is that worth mentioning?

Reply:
sure or we can do (and adjacent roles)

Reply:
Yeaa that works as well!